### PR TITLE
Submit Ramper Wallet integration

### PR DESCRIPTION
**Pull request by khiemsoft**
# Description

Please include a summary of the wallet and how users can install it. Be sure to take a look at our guide on [Custom Wallets](https://github.com/near/wallet-selector/blob/main/packages/core/docs/guides/custom-wallets.md) to understand core concepts when integrating with Wallet Selector.

# Summary
Onboard users instantly with Ramper. Ramper provides free web3 SDK for auth, key management, and crypto/NFT purchase for DApps built on multi-chains.

# Note
Please take a look at the packages. json, we need to install the ramper wallet library @ramper/near to support the ramper connector. If you have any questions, please let me know.

# Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

**Comment by kujtimprenkuSQA** [comment]
Hi @khiemsoft, thank you for submitting this PR, can you please change the base branch from `main` to `dev`?

You can find more about the process here: https://github.com/near/wallet-selector/blob/main/CONTRIBUTING.md#branching

**Comment by AmmarHumackicSQA** [comment]
Hi @khiemsoft , regarding listing criteria for wallets that you can find in our [CONTRIBUTING.md](https://github.com/near/wallet-selector/blob/main/CONTRIBUTING.md#listing-third-party-wallet-on-wallet-selector) file if you could please confirm that the criteria are met?

Here is the [link](https://github.com/near/wallet-selector/pull/407#issuecomment-1249285651) to the conversation from before regarding the other [PR](https://github.com/near/wallet-selector/pull/407) you submitted, so you can check it if something is not clear. Thanks!

**Comment by khiemsoft** [comment]
> 

Here are the updated Wallet Security Criteria for Ramper Wallet: 
https://github.com/coin98/near-ramper-wallet-security-criteria

File: packages/ramper-wallet/src/lib/ramper-wallet.ts [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+      const transactions: Array<{
+        receiverId: string;
+        actions: Array<Action>;
+      }> = await transformTransactions([{ receiverId, actions }]);
+
+      const { result } = await sendTransaction({
+        transactionActions: transactions,
+      });
+
+      return result[0];
```
[the remark this answers is not in the document]
Comment (khiemsoft) [reply packages/ramper-wallet/src/lib/ramper-wallet.ts]: Comment fixed in the latest commit. Thanks for your review.

File: packages/ramper-wallet/src/lib/injected-ramper-wallet.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import type { Transaction } from "@near-wallet-selector/core";
```
[the remark this answers is not in the document]
Comment (khiemsoft) [reply packages/ramper-wallet/src/lib/injected-ramper-wallet.ts]: Comment fixed in the latest commit. Thanks for your review.

File: packages/ramper-wallet/src/lib/ramper-wallet.ts [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+      return {
+        receiverId: transaction.receiverId || contract.contractId,
+        actions: parsedActions,
+      };
+    });
+  };
+
+  return {
+    async signIn() {
+      await signIn(); // signIn first because if getUser break function
```
[the remark this answers is not in the document]
Comment (khiemsoft) [reply packages/ramper-wallet/src/lib/ramper-wallet.ts]: Comment fixed in the latest commit. Thanks for your review.

**Review by kujtimprenkuSQA — APPROVED** [review]

**Pull request merged**